### PR TITLE
fix(tooltip): fix missing tooltip in consecutive buttons

### PR DESCRIPTION
**Pull request by wattachai-lseg**
## Description

fix tooltip bug in https://jira.refinitiv.com/browse/STG-237

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: packages/elements/src/sparkline/__test__/sparkline.test.js [hunk 1]
```diff
@@ -75,7 +75,8 @@ describe('sparkline/Sparkline', () => {
       expect(isCanvasBlank(canvas)).to.be.true;
 
       el.data = data;
-      await elementUpdated();
+      await elementUpdated(el);
+      await nextFrame(3); // ensure that the rendering's completed
```
Comment (Sakchai-Refinitiv) [inline packages/elements/src/sparkline/__test__/sparkline.test.js]: Are two frames could be enough?

File: packages/elements/src/clock/__test__/clock.test.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ describe('clock/Clock', () => {
       el = await fixture('<ef-clock analogue></ef-clock>');
       el.style.width = '129px';
       await elementUpdated(el);
-      await nextFrame();
+      await nextFrame(2); // Chrome 111 & Firefox 111 needs another frame to complete rendering
```
Comment (Sakchai-Refinitiv) [inline packages/elements/src/clock/__test__/clock.test.js]: I can confirm the native resize observer makes some test cases need one extra frame. Please update the code comments.

Comment (wattachai-lseg) [reply packages/elements/src/clock/__test__/clock.test.js]: comment updated.

File: packages/elements/src/sparkline/__test__/sparkline.test.js (hunk 1 above)
Comment (wattachai-lseg) [reply packages/elements/src/sparkline/__test__/sparkline.test.js]: Let me double check that.
Update: it actually works!

**Review by Sakchai-Refinitiv — APPROVED** [review]

**Pull request merged**